gradle.properties

```
@@ -3,19 +3,17 @@ org.gradle.jvmargs=-Xmx1G
 org.gradle.parallel=true
 
 # Mod Properties
-mod_version = 1.2.4
+mod_version=1.3.0
```

I'll be bumping as a minor update, since it's only bug fixes (from the user perspective).

```suggestion
mod_version=1.2.5
```